src/bake/bake.zig

```diff
@@ -646,6 +677,11 @@ pub const Framework = struct {
             // Support `esm-env` package using this condition.
             try out.options.conditions.appendSlice(&.{"development"});
         }
+        // Ensure "node" condition is included for server-side rendering in dev mode
+        // This helps with package.json imports field resolution
+        if (mode == .development and (renderer == .server or renderer == .ssr)) {
```

Comment:
This shouldn't just be in dev mode.
```suggestion
        if (renderer == .server or renderer == .ssr) {
```